guides/v2.1/frontend-dev-guide/translations/theme_dictionary.md

```diff
@@ -61,6 +61,15 @@ The locale dictionary would use the default values (keys) in the left column fol
 "Card Verification Number",	<translation>
 ```
 
+## Element translations per Module
+
```

Comment:
Should translating elements differently be encouraged? This could lead to an inconsistent user experience.